lib/matplotlib/tests/test_table.py

```diff
@@ -229,3 +230,11 @@ def test_table_bbox(fig_test, fig_ref):
                   loc='center',
                   bbox=Bbox.from_extents(0.1, 0.2, 0.9, 0.8)
                   )
+
+
+@check_figures_equal()
+def test_table_no_unit(fig_test, fig_ref):
```

Comment:
This is more testing that table deals with its input by casting it via `str` than asserting that it does not use unit pipeline.